src/pymodaq/daq_utils/h5exporters.py

```diff
@@ -114,8 +114,15 @@ class H5h5Exporter(H5Exporter):
     def _export_data(self, node: Node, filename: str) -> None:
         """Export an h5 node"""
         # This should allow to get the base file object
-        basefile = node._v_file
-        basefile.copy_file(dstfilename=str(filename), overwrite=False)
+        if node.backend == 'tables':
+            basefile = node.node._v_file
```

Comment:
well if you found that solution and if it works, you should put the code into the Node object. Something like:
```
def get_file(self):
    pass
```

and even a copy_file method into the Node object

you're removing the backend specifics into the backend module, which is better

Reply:
If you want but to be honest I don't like a method that returns ones of these complicated File objects from entirely different libraries.

Reply:
Then would it prove useful to create a File object wrapper. When I started this thing I moved up the hierarchy with always reasons to wrap higher objects with a common interface. In that case, it would be the File object. We don't need many methods so should be done fairly easily.

Reply:
If you think this is a good idea sure.